api/app/src/main/resources/application.properties

```diff
@@ -1,8 +1,9 @@
-spring.datasource.url=jdbc:postgresql://localhost:5432/packit
+spring.datasource.url=jdbc:postgresql://localhost:5432/packit?stringtype=unspecified
```

Comment:
Adding `stringtype=unspecified` is required for inserting json strings into the db - otherwise everything gets sent as a `varchar` which fails when it hits the db.